tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/types/UniqueGift.kt

```diff
@@ -0,0 +1,30 @@
+package dev.inmo.tgbotapi.types
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * This object describes a unique gift that was upgraded from a regular gift.
+ *
+ * @param baseName Human-readable name of the regular gift from which this unique gift was upgraded
+ * @param name Unique name of the gift. This name can be used in `https://t.me/nft/...` links and story areas
+ * @param number Unique number of the upgraded gift among gifts upgraded from the same regular gift
+ * @param model Model of the gift
```

Comment:
Sorry, but this and following params descriptions are useless. The class names already said that `model` is `UniqueGiftModel` -> `Unique model of gift`. I see two possible solutions for this problem:

* Remove useless params kdocs
* Improve params kdocs to describe how and when they will be passed